sdk/containerregistry/Azure.Containers.ContainerRegistry/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.2.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.1 (2023-12-13)
 
 ### Bugs Fixed
 
-### Other Changes
+- Fixed an issue where content client `DownloadBlobStreaming()` method would throw for blobs greater than 2GB in size ([#37113](https://github.com/Azure/azure-sdk-for-net/pull/37113)).
+- Fixed an issue in the ACR client to enable redirects due to the change in Azure.Core 1.33.0, which made redirects disabled by default ([#39377](https://github.com/Azure/azure-sdk-for-net/issues/39377)).
```

Comment:
I don't have a strong opinion one way or another, but since you linked to the PR in the comment above, it might be nice to link to the PR with the fix rather than the bug report in this entry as well.  Up to you though - I don't know that our team has a precedent we follow here for changelogs.

Reply:
We typically include a link to the corresponding issue. But I couldn't find an issue for https://github.com/Azure/azure-sdk-for-net/pull/37113, so I added the PR link since it contains the complete description [here](https://github.com/Azure/azure-sdk-for-net/pull/37113#issue-1765673857).

Reply:
@annelo-msft - Do you know if we have an issue for https://github.com/Azure/azure-sdk-for-net/pull/37113?

Reply:
Ah, I see!  Thanks for the explanation!  I don't remember offhand if that fix went in in response to a GH-filed bug report, something that came in over email.  I will do better in the future of noting this in the PR description!